docs/bot/setup_debian.md

```diff
@@ -14,106 +14,149 @@ This installation method is intended for experienced sysadmins.
 
 :::
 
-## Installation
+## Prerequisites & System User
 
-install git curl and sudo
+### Install System Packages
 
-```shell
-apt update && apt install -y git curl sudo
+Update `apt` and install required tools **as your root user**:
+
+```bash
+apt update
+apt install -y git curl sudo
 ```
 
-install node 20 from the node source repo, the full instructions can be found at https://nodesource.com/products/distributions
+- `git` : version control
+- `curl` : data transfer
+- `sudo` : execute commands as root
+
+### Create User
+
+Create the `draupnir` system user (no login shell):
+
+```bash
+useradd --system --home-dir /opt/draupnir draupnir
+```
 
-```shell
+### Install Node.js (v20)
+
+Draupnir requires Node.js 20.x. Install via NodeSource:
+
+```bash
 curl -fsSL https://deb.nodesource.com/setup_20.x -o nodesource_setup.sh
 bash nodesource_setup.sh
-apt update && apt install nodejs -y
+apt update
+apt install -y nodejs
 ```
 
-install yarn via npm
+Verify Node.js version:
 
-```shell
-npm install --global yarn
+```bash
+node --version
+# Should output v20.x.x
 ```
 
-create the directory to clone the repo
+## Install Yarn
 
-```shell
-mkdir /opt/mod-bot
+```bash
+npm install --global yarn
+yarn --version
```

Comment:
We are expecting to see 1.22.x here (yarn classic). Can we get people to use yarn via corepack instead (our package.json tells corepack which version to use)?

Reply:
You'll have to help me out a bit with corepack? I'm not that familiar with yarn.